core/controllers/collection_editor.py

```diff
@@ -69,7 +69,8 @@ def get(self, collection_id):
             'title': collection.title,
         })
 
-        self.render_template('pages/collection_editor/collection_editor.html')
+        self.render_template('pages/collection_editor/' +
```

Comment:
`collection_editor` -> `collection-editor-page`

Reply:
Ill do the renaming after the e2e tests pass as discussed in admin page PR. 
Thanks!